t/Equaldex.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Equaldex)],
+
+    'lgbt china' => test_spice(
+        '/js/spice/equaldex/china',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+
+    'lesbian rights cyprus' => test_spice(
+        '/js/spice/equaldex/cyprus',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'gay rights australia' => test_spice(
+        '/js/spice/equaldex/australia',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'transgender rights usa' => test_spice(
```

Comment:
@moollaza Yup I saw those updates! Props to @zachthompson :smile:

I remember what's causing this, USA is included in the Country package however the [CODESET](http://perldoc.perl.org/Locale/Country.html) needs to be specified as by default it matches only 2 letter codes. I'll push an update.